schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web page context",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "gdpr",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+
+    "type": "object",
+    "properties": {
+        "basisForProcessing": {
+          "enum": ["consent", "contract", "legal obligation", "vital interest", "public task", "legitimate interests"],
```

Comment:
Do we have a control over these values? I have a slight preference towards enums without spaces.